src/imitation/testing/expert_trajectories.py

```diff
@@ -0,0 +1,167 @@
+"""Test utilities to conveniently generate expert trajectories."""
+import math
+import pathlib
+import pickle
+import traceback
+import warnings
+from os import PathLike
+from pathlib import Path
+from typing import Sequence
+
+import huggingface_sb3 as hfsb3
+from filelock import FileLock
+from torch.utils import data as th_data
+
+from imitation.data import rollout, types, wrappers
+from imitation.policies import serialize
+from imitation.util import util
+
+
+def generate_expert_trajectories(
+    env_id: str,
+    num_trajectories: int,
+) -> Sequence[types.TrajectoryWithRew]:
+    """Generate expert trajectories for the given environment.
+
+    Note: will just pull a pretrained policy from the huggingface model hub.
+
+    Args:
+        env_id: The environment to generate trajectories for.
+        num_trajectories: The number of episodes to generate.
+
+    Returns:
+        A list of trajectories with rewards.
+    """
+    env = util.make_vec_env(
+        env_id,
+        post_wrappers=[lambda e, _: wrappers.RolloutInfoWrapper(e)],
+    )
+    trajectories = rollout.rollout(
+        serialize.load_policy("ppo-huggingface", env, env_name=env_id),
```

Comment:
Nit but I'd rather introduce an intermediate variable `policy` to hold this, although only one line it's doing something non-trivial here.